cinema_service/urls.py

```diff
@@ -2,10 +2,55 @@
 from django.conf.urls.static import static
 from django.contrib import admin
 from django.urls import path, include
+from drf_spectacular.views import (
+    SpectacularAPIView,
+    SpectacularSwaggerView,
+    SpectacularRedocView
+)
+from rest_framework_simplejwt.views import (
+    TokenObtainPairView,
+    TokenRefreshView
+)
+
 
 urlpatterns = [
     path("admin/", admin.site.urls),
-    path("api/cinema/", include("cinema.urls", namespace="cinema")),
-    path("api/user/", include("user.urls", namespace="user")),
+    path(
+        "api/cinema/",
+        include("cinema.urls",
+                namespace="cinema")
```

Comment:
The `include` function is used with a namespace argument in the path for 'cinema.urls'. Ensure that the 'cinema' app's URL configuration includes an `app_name` attribute to support namespacing.